haskell/doctest.bzl

```diff
@@ -187,6 +189,7 @@ def _haskell_doctest_single(target, ctx):
         # in this case.
         env = dicts.add(
             {
+                "LIBRARY_PATH": library_path,
```

Comment:
Is `LIBRARY_PATH` a magic env variable used by GHC’s internal linker? If so, it should be documented somewhere.

Reply:
It's documented in the [GHCi user-guide](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/ghci.html?highlight=library_path#extra-libraries).
Not too clearly though. In this case setting `LD_LIBRARY_PATH` alone was not sufficient.